Adding meeting notes for Editors meeting 42 (#248)
cardano-foundation/CIPs: BiweeklyMeetings/2022-04-05.md

### PR234

[CIP-47? | Proposal for open Daedalus or desktop wallet via URL](https://github.com/cardano-foundation/CIPs/pull/234)

**Sebastian Guillemot** - All right, hearing none, moving on. CIP-47? | Proposal for open Daedalus or desktop wallet via URL Triage is to basically add DApp connector to Daedalus by taking advantage of the URI scheme. To be honest, I'm not super convinced this even needs to be a CIP because the way they want to do it is very much Daedalus-specific and I'm not sure there's a way for them to not do Daedalus-specific, which is my main concern. The reason why is because for the web Cardano stuff, this works, they can use this, but Daedalus only supports a single network per installation Daedalus, and so they want to introduce a new web Cardano test net, and then they have multiple implementations of Daedalus. They want to have like a web Cardano flight for test modes. So this is kind of awkward because this won't really work with other wallets well, and it won't work with DApps well either.

**Matthias Benkort** - It's kind of weird to have a Daedalus-specific CIP in a way.

**Sebastian Guillemot** - Yeah.

> **Matthias Benkort** - The point of having a CIP would be to, well, have cost compatibility or anti rob between wallets. So if it is something Daedalus-specific, then [crosstalk 00:01:56].

Reviewer: ```suggestion
**Matthias Benkort** - The point of having a CIP would be to, well, have cost compatibility or interop between wallets. So if it is something Daedalus-specific, then [crosstalk 00:01:56].
```

> **Sebastian Guillemot** - What they can't standardize is this extension to the web Cardano stuff.

Reviewer: ```suggestion
**Sebastian Guillemot** - What they can standardize is this extension to the web Cardano stuff.
```

**Matthias Benkort** - Okay. I'm just reading through your comments also. Probably... Yeah.

**Sebastian Guillemot** - I think extending this CIP to extend to not just public keys, not just vkeys, but also native scripts is probably not too hard, and the hard part [crosstalk 00:09:21].

**Matthias Benkort** - Shouldn't it be just a hash in the end?

**Sebastian Guillemot** - Yeah. [crosstalk 00:09:26].

**Matthias Benkort** - You don't care much what's the pre images about so long as it's a valid credential. Maybe a Plutus script, native script or a vkey.

> **Sebastian Guillemot** - Yeah. So I think from the actual CDL specification from the CIP perspective, this is not a hard change. Someone just needs to make it like some sort of or with a tag. The question is just whether or not they want to make that change now or later. I suggest we at least make it now so we have backwards compatibility, and then I'll see this is something for wallets to add support and something for the Catalyst team as well to add support when they scan the chain.

Reviewer: ```suggestion
**Sebastian Guillemot** - Yeah. So I think from the actual CDDL specification from the CIP perspective, this is not a hard change. Someone just needs to make it like some sort of or with a tag. The question is just whether or not they want to make that change now or later. I suggest we at least make it now so we have backwards compatibility, and then I'll see this is something for wallets to add support and something for the Catalyst team as well to add support when they scan the chain.
```
